libs/langchain/tests/integration_tests/vectorstores/test_pgvector.py

```diff
@@ -44,6 +44,7 @@ def test_pgvector() -> None:
         embedding=FakeEmbeddingsWithAdaDimension(),
         connection_string=CONNECTION_STRING,
         pre_delete_collection=True,
+        engine_args={"pool_recycle": 3600},
```

Comment:
Why is this needed in the tests?

Reply:
Added for testing the use of the new argument, the latest commit reverts the change.